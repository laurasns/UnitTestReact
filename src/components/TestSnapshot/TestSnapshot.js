import React, { Component } from "react";

export default class TestSnapshot extends Component {
  render() {
    return (
      <div>
        <h1>Menú de Películas</h1>
        <br />
        <ul>
          <li>Jurassic Park</li>
          <li>Pesadilla en Elm Street</li>
          <li>Hermano Oso</li>
          <li>La Gran Apuesta</li>
          <li>Los Vengadores</li>
          <li>Yo, Robot</li>
        </ul>
      </div>
    );
  }
}
