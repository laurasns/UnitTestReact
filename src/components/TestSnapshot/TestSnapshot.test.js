import React from "react";
import { render } from "@testing-library/react";
import TestSnapshot from "./TestSnapshot";

test("render a snapshot", () => {
  const { baseElement } = render(<TestSnapshot />);
  expect(baseElement).toMatchSnapshot();
});
