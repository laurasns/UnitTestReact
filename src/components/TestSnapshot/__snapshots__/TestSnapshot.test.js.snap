// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a snapshot 1`] = `
<body>
  <div>
    <div>
      <h1>
        Menú de Películas
      </h1>
      <br />
      <ul>
        <li>
          Jurassic Park
        </li>
        <li>
          Pesadilla en Elm Street
        </li>
        <li>
          Hermano Oso
        </li>
        <li>
          La Gran Apuesta
        </li>
        <li>
          Los Vengadores
        </li>
        <li>
          Yo, Robot
        </li>
      </ul>
    </div>
  </div>
</body>
`;
